src/components/data/RenameColumnModal.js

```diff
@@ -0,0 +1,55 @@
+import _ from 'lodash/fp'
+import { Fragment, useState } from 'react'
+import { div, h } from 'react-hyperscript-helpers'
+import { ButtonPrimary, IdContainer, spinnerOverlay } from 'src/components/common'
+import { ValidatedInput } from 'src/components/input'
+import Modal from 'src/components/Modal'
+import { Ajax } from 'src/libs/ajax'
+import { withErrorReporting } from 'src/libs/error'
+import { FormLabel } from 'src/libs/forms'
+import * as Utils from 'src/libs/utils'
+
+
+export const tableNameInput = ({ inputProps, ...props }) => h(ValidatedInput, {
```

Comment:
What's the motivation for extracting this into a separate function? Given that this only sets a few props and is only called once, I think I'd vote to inline it.

Reply:
I'm guilty of copying this from another component. Inlined in c18c281c65133c0b6633cea508bb334896898066

Reply:
Looks like it was inlined in RenameTableModal. Did you mean to inline it here as well?

Reply:
Oh whoops- yes I did

Reply:
Inlined in the 2nd spot: d4c8621320b1efac31922b86ba8b07ec10709da4